### PR TITLE
Pull request for temporary repo pointer

### PR DESCRIPTION
**Pull request by CodeCombiner**

### CODE REVIEW

File: src/apps/App.js [hunk 1]
```diff
@@ -221,7 +221,7 @@ var App = module.exports = Class(function () {
       name: 'devkit-core',
       ssh: 'git@github.com:',
       https: 'https://github.com/',
-      repo: 'hashcube/devkit-core',
+      repo: 'CodeCombiner/devkit-core',
```
Comment (vivekb91) [inline src/apps/App.js:224]: This change is not relevant for hashcube repo

File: src/install/index.js [hunk 2]
```diff
@@ -158,7 +158,7 @@ function updateCache(name, url, version) {
  */
 function installModuleFromName (app, name, version, opts) {
   if (name == 'devkit-core') {
-    return installModuleFromURL(app, name, 'git@github:gameclosure/devkit-core', version, opts);
+    return installModuleFromURL(app, name, 'git@github:CodeCombiner/devkit-core', version, opts);
```
Comment (vivekb91) [inline src/install/index.js:161]: this also

**Comment by rampr** [comment]
My understanding is this is temporary

**Comment by CodeCombiner** [comment]
Yes but I think this pull request is incorrect. We don't need it, closing it.

**Comment by CodeCombiner** [comment]
I

**Pull request closed without merging**